pkg/kwokctl/runtime/util.go

```diff
@@ -117,8 +118,25 @@ func applyComponentPatch(component *internalversion.Component, patch internalver
 	component.Volumes = append(component.Volumes, patch.ExtraVolumes...)
 	component.Envs = append(component.Envs, patch.ExtraEnvs...)
 
+	argsmap := make(map[string]string)
+	for _, arg := range component.Args {
+		if strings.Contains(arg, "=") {
```

Comment:
```suggestion
		if !strings.Contains(arg, "=") {
```

Reply:
Thanks, seems like there have some  problem in the final push stage